src/server/raft_client.rs

```diff
@@ -16,15 +16,18 @@ use grpcio::{
 };
 use kvproto::raft_serverpb::RaftMessage;
 use kvproto::tikvpb::{BatchRaftMessage, TikvClient};
+use protobuf::Message;
 use raftstore::router::RaftStoreRouter;
 use tikv_util::collections::{HashMap, HashMapEntry};
-use tikv_util::mpsc::batch::{self, Sender as BatchSender};
+use tikv_util::mpsc::batch::{self, BatchCollector, Sender as BatchSender};
 use tikv_util::security::SecurityManager;
 use tikv_util::timer::GLOBAL_TIMER_HANDLE;
 use tokio_timer::timer::Handle;
 
 const MAX_GRPC_RECV_MSG_LEN: i32 = 10 * 1024 * 1024;
 const MAX_GRPC_SEND_MSG_LEN: i32 = 10 * 1024 * 1024;
+// When merge raft messages into a batch message, leave a buffer.
+const GRPC_SEND_MSG_BUF: usize = 1024;
```

Comment:
Is it large enough when there are many little messages?

Reply:
It's enough because merge `RaftMessage`s into `BatchRaftMessage` only adds some bytes.